### DISCUSSION TITLE
Support for custom DeepStack models

### DISCUSSION THREAD

**Discussion by 00madz** [djdd87/SynoAI#140] [Q&A]
Is there a way to setup a camera to use a custom deepstack model
https://docs.deepstack.cc/custom-models-samples/

In particular I'd like to use the license plate recognition feature:
Details on how to do that are here: https://github.com/odd86/deepstack_licenceplate_model

**Comment by djdd87** [comment]
There's been a request for face recognition as well #139 , so potentially I can do some rework to support some of the fancier parts of DeepStack.

**Reply by 00madz** [reply]
With the custom models I was hoping it'd just need the ability to specify a complete/custom URL (rather than just the deepstack server)? maybe a Deekpstack-Custom AI type? I'd be happy to throw a few more coffees your way if you get the time to look at this :D

**Reply by djdd87** [reply]
No problem, I see what needs doing now! I thought the Dark DeepStack was a separate AI entirely, but turns out it's just custom models too. If I implement #117, then this'll implement custom models for you too.

**Reply by JoeyJoe6** [reply]
following
